src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -250,13 +251,30 @@ export const WorkspaceDashboard = _.flow(
           ]),
           ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
         ]),
-        div({ style: { margin: '1.5rem 0 0.5rem 0', borderBottom: `1px solid ${colors.gray[3]}` } }),
+        div({ style: { margin: '1.5rem 0 1rem 0', borderBottom: `1px solid ${colors.gray[3]}` } }),
+        div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
+          'Google Bucket',
+          h(Clickable, {
+            as: 'span',
+            style: { margin: '0 0.5rem', color: colors.green[0] },
```

Comment:
This could use `linkButton` to get the appropriate color rather than specifying it.